ui-v2/app/components/topology-metrics/series/index.js

```diff
@@ -41,37 +41,39 @@ export default Component.extend({
     },
     change: function(evt) {
       this.data = evt.data;
-      setTimeout(()=>{
-        this.element.querySelector(".sparkline-loader").style.display = 'none';
+      setTimeout(() => {
+        this.element.querySelector('.sparkline-loader').style.display = 'none';
         this.drawGraphs();
-      }, fakeMetricsLatency())
-    }
+      }, fakeMetricsLatency());
+    },
   },
 
   drawGraphs: function() {
     if (!this.data.series) {
-      return
+      return;
     }
 
-    let svg = select(this.element.querySelector("svg.sparkline"))
+    let svg = select(this.element.querySelector('svg.sparkline'));
     svg.selectAll('path').remove();
     svg.selectAll('rect').remove();
 
-    let bb = svg.node().getBoundingClientRect()
+    let bb = svg.node().getBoundingClientRect();
     let w = bb.width;
     let h = bb.height;
 
-    if (!this.data.series) {
+    if (!this.data.series || this.data.series.length === 0) {
```

Comment:
`!this.data.series` will only check if the field is undefined, added a guard against the empty array I was getting.